### DISCUSSION TITLE
Is it appropriate to report broken links in documentation as an issue?

### DISCUSSION THREAD

**Discussion by Creidhne86** [Revolutionary-Games/Thrive#4240] [Q&A]
The issues area specifically talks about reporting technical issues, so I wanted to check here. But as I was starting to peruse the documentation before trying to contribute, I noticed the "[step by step section] https://docs.godotengine.org/en/stable/getting_started/step_by_step/intro_to_the_editor_interface.html)" link was broken in the following location: https://github.com/Revolutionary-Games/Thrive/blob/master/doc/learning_godot.md

I'm hoping to pick up an issue soon to begin contributing!

**Comment by hhyyrylainen** [comment]
Yes, broken links are technical problems and not a matter of opinion or game design.
